.editorconfig

```diff
@@ -113,3 +113,7 @@ dotnet_diagnostic.IDE1003.severity=suggestion
 dotnet_diagnostic.IDE1004.severity=suggestion
 dotnet_diagnostic.IDE1007.severity=suggestion
 dotnet_diagnostic.IDE1008.severity=suggestion
+
+ # Enforce file header
+dotnet_diagnostic.IDE0073.severity = warning
```

Comment:
I would suggest to either:
- remove this, since the `IDE0073` state that when `file_header_template` is set it requires a file header. Since we did not specify any rules explicitly in this file that should be a warning, we might as well be consistent and omit it
- Set it to `error` if we always want to enforce it.

Reply:
Fair point, I will set it to error